backend/models/user.js

```diff
@@ -169,7 +169,7 @@ User.authenticate =  async function (username, password) {
 	try {
 		await db.authenticate(user.user, password);
 	} catch (err) {
-		const resCode = utils.mongoErrorToResCode(err);
+		const resCode = err;
```

Comment:
i don't think we need to do this declaration anymore? either rename `err` to resCode` or just use err throughout?